packages/markdown/remark/src/highlight.ts

```diff
@@ -60,6 +60,10 @@ export function highlightCodeBlocks(tree: Root, highlighter: Highlighter) {
 		const html = highlighter(code, languageMatch?.[1] || 'plaintext', { meta });
 		// The replacement returns a root node with 1 child, the `<pr>` element replacement.
 		const replacement = fromHtml(html, { fragment: true }).children[0] as Element;
+
+		// Adds dataLanguage property to replacement `<pre>`
+		replacement.properties.dataLanguage = languageMatch?.[1] || "plaintext"
```

Comment:
Can we extract the `languageMatch?.[1] || "plaintext"` as a variable so it can be shared on line 60 too?